package.json

```
@@ -58,7 +58,7 @@
     "js-yaml": "^4.1.0",
     "lodash": "^4.17.20",
     "screwdriver-config-parser": "^7.5.5",
-    "screwdriver-data-schema": "^21.26.0",
+    "screwdriver-data-schema": "^21.26.3",
```

can you change it to `.4` for latest data-schema
```suggestion
    "screwdriver-data-schema": "^21.26.4",
```